tutorials/move-code-playground-tutorial-01.md

```diff
@@ -0,0 +1,331 @@
+
+## Before You Begin
+Getting oriented to the Move Code Playground is easy. With just a web browser and a few key concepts, you can use this online code editor to build, test, and run Move packages on Move compatible blockchains like Aptos.
+
+If you are not yet familiar with the Move language, the [Move Book](https://diem.github.io/move/) is a great developer resource. 
+
+For everyone else, let's check out the interface.
+
+## Navigating an Example Project
+To help developers quickly familiarize themselves with Move packages, the MCP automatically loads an example-project when you first visit the IDE. This project appears in the top left navigation pane when you click on 'Projects'.
+
+| ![Welcome to the Move Code Playground.](/assets/move-code-playground-tutorial-images/orientation.png "Move Code Playground Orientation.") |
+|:--|
+| *Fig.1 - Move Code Playground: Interface and pre-loaded example-project.* |
+
+In order to expand the file tree, simply click on the 'example-project' label within the "Explorer" panel.
+
+In this section, we'll expand the example project tree to highlight how your Move projects should be set up. We will briefly touch on the following key concepts:
+
+* The Move folder structure
+* Project dependencies
+* Addresses
+* Development reminders
+
+
+#### A Move project folder structure
+
+```
+example-project/
+  scripts/
+    file_name.move
+  sources/
+    Filename.move
+  tests/
+    FileName.move
+```
+**Within the folder structure: <br>**
+ All Modules are listed under `sources/`.<br>
+ The scripts to interact with Modules are listed under `scripts/`.<br>
+ And all test files appropriately go under `tests/`.
+
+
+
+#### Project Dependencies
+Clicking on the left navigation panel's 'Dependencies' icon reveals that the Move Code Playground comes pre-loaded with two dependencies:
+* AptosFramework - A set of modules specific to the Aptos blockchain.
+* MoveStdlib - The standard library of the Move language.
+
+*Note: When building your own projects, you may need to add these manually.*
+
+| Description       | Url                                       | Rev                           | Subdir |
+| -------------     | -------------                             |:-------------                :|--------|
+| Aptos Framework   | https://github.com/aptos-labs/aptos-core  | 367608ab5cf726039ff44bfdac1f2177210b1440     | aptos-move/framework/aptos-framework/ |
+| Move Std&nbsp;Lib | https://github.com/aptos-labs/aptos-core  | 367608ab5cf726039ff44bfdac1f2177210b1440     | aptos-move/framework/move-stdlib/     |
+
+#### Addresses
+Clicking on the left navigation panel's 'Addresses' icon reveals that the Move Code Playground comes pre-loaded with one Address:
+
+*Note: When building your own projects, you may need to add these manually.*
+
+| Name     | Address    | Description                                        |
+| ---------| -----------|----------------------------------------------------|
```

Comment:
Probably that one too, double check please.

Reply:
Updated and working now.